drillsrs/cmd/import_.py

```diff
@@ -9,6 +10,43 @@
 from drillsrs.cmd.command_base import CommandBase
 
 
+def tojson(filepath: str) -> str:
+    try:
+        from anki_export import ApkgReader
+    except ImportError:
+        print("must install anki_export package")
+        exit()
+    try:
+        from lxml import etree
+    except ImportError:
+        print("must install lxml package")
+        exit()
+    with ApkgReader(filepath) as apkg:
+        temp = apkg.export()
+    temp = temp[list(temp)[0]]
+    x = {"name": temp[1][3], "description": None, "tags": [], "cards": []}
+    counter = 1
```

Comment:
You can do without `counter` at all by using `for counter, s in enumerate(temp[1:]):`

Reply:
it's all done now
if there anything else can be imporved please tell me